app.rb

```diff
@@ -1,9 +1,29 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/message'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/messages' do
+    @peeps = Message.all
+    erb :'messages/index'
+  end
+  
+  get '/messages/new' do
+    erb :'messages/new'
+  end
+
+  post '/messages' do
+    Message.create(message: params[:message])
```

Comment:
I also found that symbols (message:) need to be used to effectively pass the data into the method. But I don't know why.